core-bundle/src/Twig/Extension/ContaoExtension.php

```diff
@@ -188,7 +189,14 @@ function (Environment $env, $context, $template, $variables = [], $withContext =
             new TwigFunction(
                 'contao_figure',
                 [FigureRuntime::class, 'renderFigure'],
-                ['is_safe' => ['html'], 'deprecated' => true],
+                [
+                    'is_safe' => ['html'],
+                    'deprecated_info' => new DeprecatedCallableInfo(
+                        'contao/core-bundle',
+                        '5.0',
+                        'The "contao_figure" function is deprecated, use the "figure" together with the "component/_figure.html.twig" component instead.',
```

Comment:
```suggestion
                        'The "contao_figure" function is deprecated, use the "figure" function together with the "component/_figure.html.twig" component instead.',
```

Is this what you meant?